CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## v0.1.3 (2021/10/04)
 
 - Class name changed from tidyframe to tibble
+- Features
+  - tibble::bind_rows
+  - tibble::bind_cols
```

Comment:
Can we instead make this:

```
* New methods
  + `.bind_rows()`
  + `.bind_cols()`
```

Reply:
Also - can you change any usages of `-` in the `CHANGELOG` to using a `*` for consistency?